src/app/shared/layout/header.component.ts

```diff
@@ -37,7 +36,7 @@ export class HeaderComponent implements OnInit {
   private readonly noButtonDialogMessage = "No, I don't wish to log out";
 
   /** Model for the displayed repository name */
-  currentRepo: string = '';
+  currentRepo = '';
```

Comment:
why remove?

Reply:
By current linting rules, the type of currentRepo can be inferred as a string just by its declaration as ''. In this case, additional type annotations do not make any difference.